experimental/framework/frs-client/src/interfaces.ts

```diff
@@ -51,4 +51,9 @@ export interface FrsResources {
     containerServices: FrsContainerServices;
 }
 
+export interface FrsAzFuncUser {
```

Comment:
Could we consider having this object be defined using `Pick` on the `FrsMember` object so we maintain a mapping between what's returned by the audience and what we pass into the `FrsAzFunctionTokenProvider`?